packages/env-utilities/README.md

```diff
@@ -0,0 +1,56 @@
+# `@skyra/env-utilities`
+
+Functional utilities for reading and parsing environmental variables, based on [Skyra](https://skyra.pw)'s internal tools.
+
+## Usage
+
+### Setup
+
+To setup dotenv, you use the `setup` function exported by the package:
+
+```typescript
+import { setup } from '@skyra/env-utilities';
+
+setup(import.meta.url);
+```
+
+You can also pass a `string` or if you want to define other options, you may use `EnvSetupOptions`. Optionally, you may configure dotenv via environment variables:
+
+- `DOTENV_DEBUG`: configures `EnvSetupOptions.debug`. If enabled, the library will log to help debug why certain keys or values are not being set as expected.
+- `DOTENV_ENCODING`: configures `EnvSetupOptions.encoding`. If set, it will specify the encoding of the files containing the environment variables
+- `DOTENV_ENV`: configures `EnvSetupOptions.env`. If set, it will specify a custon environment if `NODE_ENV` is not sufficient.
+- `DOTENV_PATH`: configures `EnvSetupOptions.path`. If set, it will specify a custom path to the file containing environment variables, useful for when they are located elsewhere.
+- `DOTENV_PREFIX`: configures `EnvSetupOptions.prefix`. If set, it will specify a required prefix for dotenv variables (e.g. `APP_`).
+
+### Typing Environment Variables
```

Comment:
Add a section that mentions how NODE_ENV is used and duplicate https://github.com/djdmbrwsk/dotenv-cra/blob/0f0b663e976996f9868f28f6b8df81402a51a1ba/README.md?plain=1#L64-L76